llvm/lib/Transforms/Scalar/NewGVN.cpp

```diff
@@ -3021,6 +3184,17 @@ std::pair<unsigned, unsigned> NewGVN::assignDFSNumbers(BasicBlock *B,
   return std::make_pair(Start, End);
 }
 
+unsigned NewGVN::updateDFSNumbers(unsigned ICount) {
```

Comment:
I'd make the param `unsigned &ICount` so that callers can't forget to reassign the return value to `ICount`, then for the call site where you pass in `0`, set `ICount` to `0` instead and pass that in

Reply:
Good catch! I updated the code :)